src/app/components/technical-record/technical-record.component.scss

```diff
@@ -150,7 +151,7 @@ dd {
 
 .title {
   color: $govuk-text-colour;
-  font-family: $font-stack;
+  font-family: "GDS Transport",Arial,sans-serif;
```

Comment:
as per spec font family should be GDSTransportWebsite
![image](https://user-images.githubusercontent.com/52704773/67954433-e08f8f80-fbe8-11e9-8a33-c417f4ddbdb9.png)
Any other aspect need to be handled through specificity